packages/backend/src/core/transaction-count/RpcTransactionUpdater.ts

```diff
@@ -26,18 +23,19 @@ export class RpcTransactionUpdater implements TransactionCounter {
     this.updateBlock.bind(this),
     this.logger,
     {
-      workers: 100,
+      workers: this.config.rpcWorkQueueWorkers,
       id: 'RpcTransactionUpdater.blockQueue',
     },
   )
-  private assessCount: AssessCount
-  private startBlock: number
+  private readonly assessCount: AssessCount
+  private readonly startBlock: number
 
   constructor(
-    private ethereumClient: EthereumClient,
-    private rpcTransactionCountRepository: RpcTransactionCountRepository,
-    private clock: Clock,
-    private logger: Logger,
+    private readonly config: TransactionCountSyncConfig,
```

Comment:
Why passing a separate object instead of adding new parameters to `opts`? Changing the signature requires us to update more files to add this change. What is more, many keys from `TransactionCountSyncConfig` are not used in this class.
This applies to other updaters as well.